homeassistant/components/climate/evohome.py

```diff
@@ -13,29 +13,35 @@
 from requests.exceptions import HTTPError
 
 from homeassistant.components.climate import (
-    ClimateDevice,
-    STATE_AUTO,
-    STATE_ECO,
-    STATE_OFF,
-    SUPPORT_OPERATION_MODE,
+    STATE_AUTO, STATE_ECO, STATE_MANUAL, STATE_OFF,
     SUPPORT_AWAY_MODE,
-)
-from homeassistant.components.evohome import (
-    CONF_LOCATION_IDX,
-    DATA_EVOHOME,
-    MAX_TEMP,
-    MIN_TEMP,
-    SCAN_INTERVAL_MAX
+    SUPPORT_ON_OFF,
+    SUPPORT_OPERATION_MODE,
+    SUPPORT_TARGET_TEMPERATURE,
+    ClimateDevice
 )
 from homeassistant.const import (
     CONF_SCAN_INTERVAL,
-    PRECISION_TENTHS,
-    TEMP_CELSIUS,
     HTTP_TOO_MANY_REQUESTS,
+    PRECISION_TENTHS,
+    TEMP_CELSIUS
+)
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import (
+    async_dispatcher_send,
+    async_dispatcher_connect
 )
+
 _LOGGER = logging.getLogger(__name__)
 
-# these are for the controller's opmode/state and the zone's state
+DOMAIN = 'evohome'
```

Comment:
This platform belongs to the climate domain. We're not allowed to overwrite that. We can import the evohome domain as another name.

Reply:
Actually, this `DOMAIN` is no longer needed in the platform.